order/views.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+from django.views import View
+from django.http import JsonResponse
+from django.db import transaction
+
+from cart.models import Cart
+from .models import Order, OrderItem, OrderStatus
+
+from user.utils import login_decorator
+
+class OrderpageView(View):
+    @login_decorator
+    def get(self, request):
+        try:
+            user = request.user
+            
+            user_list = [{
+                'name'    : user.name,
+                'phone'   : user.phone,
+                'address' : user.address
+            }]
+            return JsonResponse({'user' : user_list}, status=200)
+
+        except KeyError:
+            JsonResponse({'message' : 'KEY_ERROR'}, status=400)
+
+class PaymentView(View):
+    @login_decorator
+    def get(self, request):
+        try:
+            user = request.user
+            
+            total = 0
+
+            for a in user.cart_set.all():
+
+                total += a.ingredient.price * a.count
+
+            payment_list = [{
+                'name'    : user.name,
+                'address' : user.address,
+                'price'   : total
+            }]
+
+            return JsonResponse({'payment' : payment_list}, status=200)
+
+        except KeyError:
+            JsonResponse({'message' : 'KEY_ERROR'}, status=400)
```

Comment:
total_price는 프론트에서 이미 카트에 담긴 상품과 각가의 수량을 알고 있기 때문에, 불필요한 통신 없이 간단하게 구할 수 있습니다.
그러므로 아래 get method 자체가 필요없고, 유저 정보는 /users에서 호출하도록 수정하시면 좋을 것 같습니다!

```suggestion
class PaymentView(View):
    @login_decorator
```